packages/ckit/src/tx-builders/MintSudtBuilder.ts

```diff
@@ -29,9 +37,194 @@ export interface MintOptions {
 }
 
 export class MintSudtBuilder implements TransactionBuilder {
-  constructor(private options: MintOptions, private provider: MercuryProvider, private signer: Signer) {}
+  sudtTypeDep: CellDep;
+  pwLockDep: CellDep;
+  acpLockDep: CellDep;
+  constructor(private options: MintOptions, private provider: MercuryProvider, private signer: Signer) {
```

Comment:
```suggestion
  constructor(private options: MintOptions, private provider: CkitProvider, private signer: Signer) {
```
`CkitProvider` would be more helpful when `getScriptConfig`